### PR TITLE
Add python-oifits to conda-forge

### PR DESCRIPTION
**Pull request by shreyasbapat**
Checklist for `oifits` package

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by shreyasbapat** [comment]
@conda-forge/help-python 

Can you people please help?

File: recipes/oifits/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  sha256: 611d8ef9040aabf9db87e9419a25be18350f59341771587dde3d6d3cafc46f48
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```
Comment (scopatz) [inline recipes/oifits/meta.yaml]: this isn't noarch if it needs a compiler

Comment (shreyasbapat) [reply recipes/oifits/meta.yaml]: Thanks @scopatz ! I fixed it!

**Comment by shreyasbapat** [comment]
@conda-forge/help-python Done!

**Pull request merged**